test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj

```diff
@@ -189,125 +192,139 @@
 
 (defn get-webserver-context-for-server
   []
-  {:state (atom nil)
-   :handlers
-           (ContextHandlerCollection.)
-   :server nil})
-
-(defn get-http-config-for-server
-  [max-threads queue-max-size so-linger-milliseconds idle-timeout-milliseconds]
-  {:http {:port 0
-          :host "0.0.0.0"
-          :request-header-max-size 100
-          :so-linger-milliseconds so-linger-milliseconds
-          :idle-timeout-milliseconds idle-timeout-milliseconds}
-   :jmx-enable false
-   :max-threads max-threads
-   :queue-max-size queue-max-size})
-
-(defn get-thread-pool-and-queue-for-create-server
```

Comment:
The guts of this moved over to `get-server-thread-pool-queue` in the jetty9-default-config-test namespace, since it now has a similar need for this.